bedrock/base/templates/macros.html

```diff
@@ -558,38 +553,48 @@
 
+              {% if activesub in ('index', 'trust', 'customize', 'fast') %}
               <ul class="subsubnav" id="desktop-subsubnav">
-                <li><a href="{{ url('firefox.desktop.trust') }}" data-id="desktop-trust">{{ _('Trusted') }}</a></li>
-                <li><a href="{{ url('firefox.desktop.customize') }}" data-id="desktop-customize">{{ _('Flexible') }}</a></li>
-                <li><a href="{{ url('firefox.desktop.fast') }}" data-id="desktop-fast">{{ _('Fast') }}</a></li>
+                <li><a href="{{ url('firefox.desktop.trust') }}"{% if activesub == 'trust' %} class="selected"{% endif %}>{{ _('Trusted') }}</a></li>
+                <li><a href="{{ url('firefox.desktop.customize') }}"{% if activesub == 'customize' %} class="selected"{% endif %}>{{ _('Flexible') }}</a></li>
+                <li><a href="{{ url('firefox.desktop.fast') }}"{% if activesub == 'fast' %} class="selected"{% endif %}>{{ _('Fast') }}</a></li>
               </ul>
+              {% endif %}
             </li>
-            <li><a href="{{ url('firefox.sync') }}" data-id="desktop-sync"><div>{{ _('Sync') }}</div></a></li>
-            <li><a href="{{ url('firefox.hello') }}" data-id="desktop-hello"><div>{{ _('Hello') }}</div></a></li>
+            <li><a href="{{ url('firefox.sync') }}"{% if activesub == 'sync' %} class="selected"{% endif %}><div>{{ _('Sync') }}</div></a></li>
+            <li><a href="{{ url('firefox.hello') }}"{% if activesub == 'hello' %} class="selected"{% endif %}><div>{{ _('Hello') }}</div></a></li>
           </ul>
         </li>
-        <li>
-          <a href="{{ url('firefox.android.index') }}" class="primary-link" data-id="android"><div>{{ _('<span>for</span> Android') }}</div></a>
+        <li{% if activeprimary == 'android' %} class="active"{% endif %}>
+          <a href="{{ url('firefox.android.index') }}" class="primary-link"><div>{{ _('<span>for</span> Android') }}</div></a>
 
           <ul class="subnav" id="android-subnav">
-            <li><a href="{{ url('firefox.android.index') }}" data-id="android-index"><div>{{ _('Overview') }}</div></a></li>
-            <li><a href="{{ url('firefox.sync') }}" data-id="android-sync"><div>{{ _('Sync') }}</div></a></li>
+            <li><a href="{{ url('firefox.android.index') }}"{% if activesub == 'index' %} class="selected"{% endif %}><div>{{ _('Overview') }}</div></a></li>
+            <li><a href="{{ url('firefox.sync') }}"{% if activesub == 'sync' %} class="selected"{% endif %}><div>{{ _('Sync') }}</div></a></li>
           </ul>
         </li>
-        <li>
-          <a href="{{ url('firefox.os.index') }}" class="primary-link" data-id="os"><div>{{ _('OS') }}</div></a>
+        <li{% if activeprimary == 'os' %} class="active"{% endif %}>
+          <a href="{{ url('firefox.os.index') }}" class="primary-link"{% if activesub == 'index' %} class="selected"{% endif %}>
+            <div>
+            {% if l10n_has_tag('fx_family_nav_update') %}
```

Comment:
Let's just put `{{ _('Firefox OS') }}`here without the conditional, I don't plan to expose the string for translation (inside the product the brand "Firefox OS" is not localizable anyway).